proposals/wac-ucr/index.bs

```diff
@@ -1072,6 +1072,18 @@ the [=resource=] and needing administrator assistance to recover.
 
 ## Capabilities ## {#uc-capabilities}
 
+### Possesion of a group membership verifiable credential ### {#group-membership-vc}
+
+Omni Inc. granted Acme Inc. read-write access to their online store front.
+This grant allows all Acme Inc. to have that access as long as they can
+present membership credential issued by Acme Inc. At the time of the grant
+Omni Inc. notified Acme Inc. about that grant and all access details,
+including membership credential requirement for Acme Inc. members.
+Alice who works for Acme Inc. now can access Omni Inc. onlne store front
+using her membership credential issued by Acme Inc. Acme Inc. doesn't maintain
+public list of their members, and Omni Inc. can not know who is Acme Inc.
```

Comment:
```suggestion
Omni Corp. granted Acme Corp. read-write access to their online store front.
This grant allows all Acme Corp. to have that access as long as they can
present membership credential issued by Acme Corp. At the time of the grant
Omni Corp. notified Acme Corp. about that grant and all access details,
including membership credential requirement for Acme Corp. members.
Alice who works for Acme Corp. now can access Omni Corp. online store front
using her membership credential issued by Acme Corp. Acme Corp. doesn't maintain
public list of their members, and Omni Corp. can not know who is Acme Corp.
```
* "Inc." always requires a preceding comma, and often requires a trailing comma. "Corp." requires neither, thus is an easier substitution.
* also fixed typo

Reply:
thank you!